### DISCUSSION TITLE
Solar energy consumed

### DISCUSSION THREAD

**Discussion by FlokkieB** [CJNE/ha-myenergi#581] [Q&A]
Is there a sensor that shows the solar energy consumed in a charging session?

**Comment by G6EJD** [comment]
Did you look? - EV Green Energy Today

**Comment by FlokkieB** [comment]
Yes i did look. And Green Energy Today is the most likely sensor, however it continues calculating/collecting even when its dark…so my solar energy consumed for charging is a to high. I have sensors for import, export etc those are pretty ok. A bit to high compared with the solaredge measurements. I am considering to setup a calculation where the solar W import is calculated when charging. Not sure yet how to calculate import for a session. Daily, weekly erc is easy. I am using mode eco for charging.

**Comment by FlokkieB** [comment]
And btw, in the the my energy app the measurements are ok. There I see almost correct calculations (with the same difference with solaredge measurement. Those measurements are in the high level solar production the highest. Dont understand that eather. Could be a calibration issue.

**Comment by G6EJD** [comment]
If collecting solar energy after dark then the Solar CT must be in the wrong cable, so an installation error.

**Comment by FlokkieB** [comment]
Installation is Ok i think. On the Zappi i see correct generation and consumption. Checking the power sensor myenergy power generation, that is now about 0 W But daily green energy is still collecting. In the app i see also correct data.

**Comment by G6EJD** [comment]
Is more likely you still have the wrong entity, or have made some sensor amendments such as data types or there is an installation error. 
it may be easier to remove the integration, then delete all associated entities then reinstall. It’s definitely a problem with your setup.

**Comment by G6EJD** [comment]
You can easily test the entity by simply unclamping the Solar CT and if that’s the cause the increases will stop.
